docs/getting-started-guides/windows/index.md

```diff
@@ -1,149 +1,256 @@
 ---
-title: Windows Server Containers
+title: Using Windows Server Containers in Kubernetes
 ---
-**Note:** These instructions were recently updated based on Windows Server platform enhancements
+**Note:** These instructions were recently updated based on Windows Server platform enhancements and the Kubernetes v1.9 release
 
-Kubernetes version 1.5 introduced support for Windows Server Containers based on the Windows Server 2016 OS. With the release of Windows Server version 1709 and using Kubernetes v1.8 users are able to deploy a K8s cluster either on-premises or in a private/public cloud using a number of different network toplogies and CNI plugins. Platform improvements include:
-- Improved support for Pods! Shared network namespace (compartment) with multiple Windows Server containers (shared kernel)
-- Reduced network complexity by using a single network endpoint per Pod
+Kubernetes version 1.5 introduced Alpha support for Windows Server Containers based on the Windows Server 2016 operating system. With the release of Windows Server version 1709 and using Kubernetes v1.9 users are able to deploy a Kubernetes cluster either on-premises or in a private/public cloud using a number of different network topologies and CNI plugins. Some key feature improvements for Windows Server Containers on Kubernetes include:
+- Improved support for pods! Shared network namespace (compartment) with multiple Windows Server containers (shared kernel)
+- Reduced network complexity by using a single network endpoint per pod
 - Kernel-Based load-balancing using the Virtual Filtering Platform (VFP) Hyper-v Switch Extension (analogous to Linux iptables)
+- Container Runtime Interface (CRI) pod and node level statistics
+- Support for kubeadm commands to add Windows Server nodes to a Kubernetes environment
 
- The Kubernetes control plane (API Server, Scheduler, Controller Manager, etc) continue to run on Linux, while the kubelet and kube-proxy can be run on Windows Server version 1709.
+The Kubernetes control plane (API Server, Scheduler, Controller Manager, etc) continue to run on Linux, while the kubelet and kube-proxy can be run on Windows Server 2016 or later
```

Comment:
Are we say WS2016 due to the fact that OVS/OVN only require this version of the platform? Or, that initial support will continue to work on WS106?

I'm worried that this will confuse the message above about new platform support with Windows Server version 1709.

Reply:
OVS/OVN requires Windows Server 2016 or later, that means it also works with Windows Server version 1709.
As far as I know the initial support is still there and should continue working, but @bsteciuk might be a better person to ask for this.

Reply:
@JMesser81  we will continue to support customers that don't want to take a dependency on 1709. we will however stress that 1709 offers significant benefits and that's our recommendation for OS  release